config/routes.rb

```diff
@@ -1,5 +1,14 @@
 Rails.application.routes.draw do
-  get 'welcome/index'
 
+  resources :sessions, only: [:new, :create, :destroy]
+  get '/login', to: "sessions#new"
+  post '/login', to: "sessions#create"
+  delete '/logout', to: "sessions#destroy"
+
+  resources :users, except: [:index]
+
+  get 'welcome/index'
```

Comment:
@katiekeel @yoasyo25 I think line 10 was introduced as a bug; seems like root is already set. I think we should remove line 10.

Reply:
I agree.